shoop/utils/models.py

```diff
@@ -11,6 +11,7 @@
 
 from django.db import models
 from django_countries.fields import Country
+from jinja2.filters import do_batch
```

Comment:
Using jinja2 filter for this does not feel right. How about using `shoop.utils.iterables.batch` instead?